indykite/resource_ingest_mapping.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	ingestMappingJSONConfigKey = "json_config"
+	ingestMappingJSONConfigKey = "json"
```

Comment:
Why you are changing also IngestMapping? Ingest has different way of configuration than AuthZ.